src/services/llms/openai.py

```diff
@@ -23,6 +23,7 @@ def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) ->
         - Use GitHub Markdown in comments
         - Focus on bugs, security issues, and performance problems
         - IMPORTANT: NEVER suggest adding comments to the code
```

Comment:
This line introduces a potential security vulnerability.  Using `Config.HUMAN_LANGUAGE` directly in the string interpolation could lead to issues if `Config.HUMAN_LANGUAGE` is not properly sanitized and contains malicious code or unexpected characters.  Consider using a more secure method of integrating the human language, such as escaping special characters or using a template engine that handles escaping automatically.  The current approach might make your application vulnerable to injection attacks.  Please review the security implications and mitigate potential risks.